packages/amazonq/test/e2e/amazonq/VET.test.ts

```diff
@@ -1,60 +1,85 @@
-/*!
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0
- */
+import { Workbench, By, WebviewView, WebElement } from 'vscode-extension-tester'
+import { until } from 'selenium-webdriver'
 
-import { Workbench, By, EditorView, WebviewView, WebElement } from 'vscode-extension-tester'
-
-describe('Amazon Q Login Flow', function () {
+describe('Amazon Q E2E UI Test', function () {
     // need this timeout because Amazon Q takes awhile to load
-    this.timeout(30000)
+    this.timeout(150000)
```

Comment:
I think these changes are not meant to be part of this CR? They look like the ones Laura made in the other open one.